### PR TITLE
[JV2M-223] Setting up basic tests

### PR DESCRIPTION
**Pull request by davodesign84**
Basic test setup

### CODE REVIEW

**Comment by davodesign84** [comment]
@Tristyn @callensm @jetjake this PR should now be ready for a review and if you could also kick the tyres locally would be nice!

I'd like to cleanup a bunch of lint warnings as well but was thinking to have a separate task / PR for it as some might require small refactoring and I'd like to add tests before doing that. 

Let me know what you think! Next I'll get it to run all the checks as part of the CI workflow.

File: src/components/__test__/Loader.test.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { Loader } from '../Loader';
+import { render, screen } from '@testing-library/react';
+
+describe('Components - Info', () => {
+  it('should render without error', () => {
+    render(<Loader />);
+    const logo = screen.getByRole('img');
+    expect(logo).toBeInTheDocument();
+    expect(logo).toHaveProperty('alt', 'Jet Logomark');
+  });
```
Comment (Tristyn) [inline src/components/__test__/Loader.test.tsx:10]: This kind of test writing is sweet

A goal for v2 is to send a transaction from the frontend and see if it succeeds. It seems possible to add these with the testing you've set up

Comment (davodesign84) [reply src/components/__test__/Loader.test.tsx:10]: Testing Library React is pretty good in mimicking user behaviour like that. It won't handle extensions of course but we should be able to setup some headless wallet somehow (still have to look into this side of the Solana ecosystem :D)

**Review by Tristyn — APPROVED** [review]
This is an awesome testing framework. With this we'll be able to test transactions through the frontend if we mock the wallet

**Pull request merged**